src/modules/mod.rs

```diff
@@ -8,6 +8,7 @@ mod git_status;
 mod golang;
 mod jobs;
 mod line_break;
+mod nim;
```

Comment:
You have not updated const ALL_MODULES_LIST present in src/module.rs according to this update